docs/pages/index.mdx

```diff
@@ -6,102 +6,72 @@ layout: tocless-doc
 videoBanner: fSgCnQrNu78
 ---
 
-Teleport is a certificate authority and access plane for your infrastructure.
-With Teleport you can:
+Teleport is Identity-Native Infrastructure Access for engineers and machines. With Teleport you can:
 
-- Use a single solution to access your SSH servers, Kubernetes clusters,
-  databases, desktops, and web applications. 
-- Define sophisticated access policies for every component of your infrastructure, with fine-grained audit logs and session recordings.
-- Automatically on– and off-board users via integrations with single sign-on
-  providers like GitHub, Okta, and Google Workspace.
+- Replace insecure secrets with true identity.
+- Have a single solution for SSH, Kubernetes, databases, desktops, web applications, and more.
+- Define RBAC for every component of your infrastructure.
+- Gather audit log events and session recordings.
```

Comment:
I think we can expand this a bit to describe the benefit of these things. Also, I think we should say "audit events" instead of "audit log events", which sounds like the log itself is the event.